bundle/Form/Type/Content/Translation/AlwaysAvailableUpdateType.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Type\Content\Translation;
+
+use Ibexa\AdminUi\Form\Type\Content\ContentInfoType;
+use Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Data\Content\ContentUpdateAlwaysAvailableData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AlwaysAvailableUpdateType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add(
+            'content_info',
+            ContentInfoType::class,
+            [
+                'label' => false,
+                'attr' => [
+                    'hidden' => true,
+                ],
+            ],
+        );
+
+        $builder->add(
+            'always_available',
+            CheckboxType::class,
+            [
+                'label' => false,
+                'required' => false,
+                'attr' => [
+                    'hidden' => true,
+                ],
+            ],
+        );
+
+        $builder->add(
```

Comment:
Submit button should ideally be inside the template, so it can easily be styled.

Reply:
This form is actually always hidden. I'm not sure if the submit is needed at all, as the form is submitted by JS, but this copies the approach from other similar functionality from vendor, so I would leave it as it.

Reply:
As you wish :+1: